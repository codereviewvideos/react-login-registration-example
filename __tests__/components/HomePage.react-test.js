import React from 'react';
import HomePage from '../../src/components/AboutPage.react';
import renderer from 'react-test-renderer';

describe('HomePage', () => {
  it('renders as expected', () => {
    const component = renderer.create(
      <HomePage />
    );
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
