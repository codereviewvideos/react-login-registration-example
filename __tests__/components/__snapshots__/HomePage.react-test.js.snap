exports[`HomePage renders as expected 1`] = `
<div>
  <h1>
    React Slingshot
  </h1>
  <h2>
    Get Started
  </h2>
  <ol>
    <li>
      Review the 
      <a
        onClick={[Function]}
        style={Object {}}>
        demo app
      </a>
    </li>
    <li>
      Remove the demo and start coding: npm run remove-demo
    </li>
  </ol>
</div>
`;
